src/atomate2/vasp/jobs/lobster.py

```diff
@@ -0,0 +1,301 @@
+        vaspmaker: BaseVaspMaker.
+        address_max_basis: string to yaml file including basis set information.
+        address_min_basis: string to yaml file including basis set information.
+
+    Returns:
+        dict including number of bands and basis set information
+    """
+    potcar_symbols = vaspmaker.input_set_generator._get_potcar(
+        structure=structure, potcar_spec=True
+    )
+
+    # get data from Lobsterinput
+    # Lobster
+    if address_max_basis is None and address_min_basis is None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure, potcar_symbols=potcar_symbols
+        )
+    elif address_max_basis is not None and address_min_basis is None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure,
+            potcar_symbols=potcar_symbols,
+            address_basis_file_max=address_max_basis,
+        )
+    elif address_min_basis is not None and address_max_basis is None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure,
+            potcar_symbols=potcar_symbols,
+            address_basis_file_min=address_min_basis,
+        )
+    elif address_min_basis is not None and address_max_basis is not None:
+        list_basis_dict = Lobsterin.get_all_possible_basis_functions(
+            structure=structure,
+            potcar_symbols=potcar_symbols,
+            address_basis_file_max=address_max_basis,
+            address_basis_file_min=address_min_basis,
+        )
+
+    nband_list = []
+    for dict_for_basis in list_basis_dict:
+        basis = [key + " " + value for key, value in dict_for_basis.items()]
+        lobsterin = Lobsterin(settingsdict={"basisfunctions": basis})
+        nbands = lobsterin._get_nbands(structure=structure)
+        nband_list.append(nbands)
+
+    return {"nbands": max(nband_list), "basis_dict": list_basis_dict}
+
+
+@job
+def update_user_incar_settings_maker(
+    vaspmaker: VaspLobsterMaker,
+    nbands: dict,
+    structure: Structure,
+    prev_vasp_dir: Path | str,
+):
+    """
+    Update the INCAR settings of a maker
+    Args:
+        vaspmaker: VaspLobsterMaker.
+        nbands: dict including "nbands"
+        structure: Structure object.
+        prev_vasp_dir: Path or string to vasp files.
+
+    Returns:
+        VaspLobsterMaker with correct number of bands.
+    """
+    vaspmaker = update_user_incar_settings(vaspmaker, {"NBANDS": nbands["nbands"]})
+    vaspjob = vaspmaker.make(structure=structure, prev_vasp_dir=prev_vasp_dir)
+
+    return Response(replace=vaspjob)
+
+
+@job
+def get_lobster_jobs(
+    basis_dict,
+    wavefunction_dir,
+    user_lobsterin_settings,
+    additional_outputs,
+    optimization_run_job_dir,
+    optimization_run_uuid,
+    static_run_job_dir,
+    static_run_uuid,
+    additional_static_run_job_dir,
+    additional_static_run_uuid,
+):
+    """
+
```

Comment:
Also numpy style docstrings here also.

Reply:
Sorry to be a pain, but there is a slight issue with this and some other docstrings.  The format should be like:

```python
    """
    Parameters
    ----------
    name: type
        Description.
    """
```

Reply:
Thanks. It's very helpful that you check so carefully!